yang/frr-vrf.yang

```diff
@@ -36,29 +40,32 @@ module frr-vrf {
           "VRF name.";
       }
 
-      leaf id {
-        type uint32 {
-          range "0..4294967295";
+      container netns {
+        /* FIXME: libyang support is required */
+        /* if-feature "netns"; */
```

Comment:
nit: to be precise, it's the FRR NB that needs some additional work to support YANG features (libyang is fine). For now, not using `if-feature` shouldn't be a problem (we can reject netns config changes in the appropriate northbound callbacks).